test/ejabberd_tests/tests/privacy_SUITE.erl

```diff
@@ -711,6 +711,7 @@ block_jid_message_but_not_presence(Config) ->
         end).
 
 newly_blocked_presense_jid_by_new_list(Config) ->
+    reset_users(Config),
```

Comment:
The only catch with that is that it creates new usernames as well, so the ones hardcoded into the privacy_helper lists don't work. Still, it feels like a neater solution, so I'll sort it out.